puppet/kandra/manifests/app_frontend_monitoring.pp

```diff
@@ -12,6 +12,6 @@
     owner   => 'root',
     group   => 'root',
     mode    => '0644',
-    source  => 'puppet:///modules/zulip/cron.d/rabbitmq-monitoring',
+    source  => 'puppet:///modules/kandra/cron.d/rabbitmq-monitoring',
```

Comment:
Hmm. We document the monitoring stuff that depends on this cron job in `troubleshooting.md`; I think we might have broken that documentation previously though... but that would be a reason for this to live outside the `kandra` directory? Not sure.

Reply:
We are already specific that those are in `puppet/kandra`:
```
The complete Nagios configuration (sans secret keys) used to
monitor zulip.com is available under `puppet/kandra` in the
Zulip Git repository (those files are not installed in the release
tarballs).

The Nagios plugins used by that configuration are installed
automatically by the Zulip installation process in subdirectories
under `/usr/lib/nagios/plugins/`. The following is a summary of the
useful Nagios plugins included with Zulip and what they check:
```